### ISSUE TITLE
ergoCub 1.3 S/N:002 – Robot doesn't start due to left wrist hardware fault

### ISSUE THREAD

**Issue by vigisushrutha23** [robotology/icub-tech-support#1895]
### Robot Name 🤖

ergoCub 1.3 S/N:002

### Request/Failure description

When using the command YARP_FORWARD_LOG_ENABLE=1 yarprobotinterface --config ergocub.xml --enable_tags "(enable_ros2)" , the robot fails to start as there is an hardware fault on the left wrist.

### Detailed context

![Screenshot from 2024-09-06 11-30-03](https://github.com/user-attachments/assets/093fccb1-b6bc-4e4d-b1d0-0521c2f7a1b8)

Upon running motorgui during startup we see the hardware fault on the left wrist showing error in the encoders.  

After a few restarts we started getting the following error as can be seen in the log.
[log_ergocub-torso_yarprobotinterface_3115.txt](https://github.com/user-attachments/files/16906248/log_ergocub-torso_yarprobotinterface_3115.txt)

### How does it affect you?

We are unable to perform experiments for an upcoming deadline.

**Comment by vigisushrutha23** [comment]
Here is another log taken after another restart reproducing the HW fault on the left wrist. 
[log_ergocub-torso_yarprobotinterface_1866.txt](https://github.com/user-attachments/files/16906463/log_ergocub-torso_yarprobotinterface_1866.txt)

**Comment by vigisushrutha23** [comment]
Following @AntonioConsilvio 's advice, we checked if the left wrist connectors were disconnected. With @mebbaid we made the connectionand the robot was able to start up. But one motion of the hands caused disconnection once again, as seen in the picture below. 

![image](https://github.com/user-attachments/assets/78d6a5d6-9ea6-49d0-93d9-a4747aa6b6aa)

After few tries and pushing the cable behind the parallel mechanism of the wrist to prevent it from disconnecting, we succeeded and we were able to run experiments. I shall leave the issue open for further verification and solidifying the connection.
